GiniVision/Classes/Core/Camera.swift

```diff
@@ -99,7 +99,7 @@ internal class Camera: NSObject {
             self.stillImageOutput?
                 .captureStillImageAsynchronously(from: connection) { (buffer: CMSampleBuffer?, error: Error?) in
                     guard error != nil else {
-                        completion(AVCaptureStillImageOutput.jpegStillImageNSDataRepresentation(buffer), nil)
+                        completion(AVCaptureStillImageOutput.jpegStillImageNSDataRepresentation(buffer!), nil)
```

Comment:
I'd use the guard that's already here to also unwrap the buffer.
I also think that the iOS SDK has many APIs where not having an error object doesn't mean everything went smoothly. I'm not sure this is the case here, but I think it would have been cleaner if you used the guard.